R/set_spark_home_env_var.R

```diff
@@ -0,0 +1,27 @@
+#' Set the SPARK_HOME environment variable
+#' 
+#' Set the \code{SPARK_HOME} environment variable. This slightly speeds up some 
+#' operations, including the connection time.
+#' @param path A string containing the path to the installation location of
+#' Spark. If \code{NULL}, the path to the most latest Spark/Hadoop versions is
+#' used. 
+#' @param verbose Logical. Should the function explain what is it doing?
+#' @return The function is mostly invoked for the side-effect of setting the 
+#' \code{SPARK_HOME} environment variable. It also returns \code{TRUE} if the 
+#' environment was successfully set, nad \code{FALSE} otherwise.
+#' @examples
+#' \dontrun{
+#' # Not run due to side-effects
+#' set_spark_home_env_var()
+#' }
+#' @export
+set_spark_home_env_var <- function(path = NULL, verbose = is.null(path)) {
```

Comment:
Nit: Since you are creating a new file, it would make sense to name it `spark_home.R` and potentially also move the `spark_home_dir` function into this file, up to you, otherwise we can move that later.